SAMv1.tex

```diff
@@ -315,7 +315,7 @@ \subsection{The header section}
   & {\tt KS} & The array of nucleotide bases that correspond to the key sequence of each read.\\\cline{2-3}
   & {\tt LB} & Library.\\\cline{2-3}
   & {\tt PG} & Programs used for processing the read group.\\\cline{2-3}
-  & {\tt PI} & Predicted median insert size.\\\cline{2-3}
+  & {\tt PI} & Predicted median insert size. \emph{Valid values}: Integers matching \verb"/^[0-9]+$/"\\\cline{2-3}
```

Comment:
The description should also be updated, e.g., to _rounded_ predicted median insert size. Otherwise, further text is owed for as to why it's an integer rather than a decimal value.